capture/session.c

```diff
@@ -26,6 +26,7 @@ extern time_t                lastPacketSecs[MOLOCH_MAX_PACKET_THREADS];
 /******************************************************************************/
 
 LOCAL int                   protocolField;
+extern uint32_t              hashSalt;
```

Comment:
make sure the h lines up with the above p, and since you'll be moving the hashSalt init to session.c change extern to LOCAL